statistics/cmsketch.go

```diff
@@ -44,29 +57,213 @@ func NewCMSketch(d, w int32) *CMSketch {
 	return &CMSketch{depth: d, width: w, table: tbl}
 }
 
+// NewCMSketchWithTopN returns a new CM sketch with TopN elements.
+// Total is the size of the whole dataset
+// Only when the number of topn elements exceeded len(data) / topnThreshold will store topn elements
```

Comment:
What does this comment mean?

Reply:
If the original data is not skew enough, then we won;t build map for top n elements.